src/components/cards/TaskCards.tsx

```diff
@@ -15,7 +15,7 @@ export default function TaskCards() {
 
   return (
     <>
-      <List className="tasks-list">
+      <List sx={{ width: 600, maxWidth: '80vw', m: 'auto' }}>
```

Comment:
you can make a separate file for styles, where you can keep your objects with styles and export them here.